lib/core/src/lib/form/components/widgets/core/form-field.model.spec.ts

```diff
@@ -112,67 +112,95 @@ describe('FormFieldModel', () => {
+                options: [{ id: 'one', name: 'One' }],
+                value: null,
+                hasEmptyValue: true
+            });
 
-    it('should set hasEmptyValue to true if "empty" option is present in options', () => {
-        const field = new FormFieldModel(new FormModel(), {
-            type: FormFieldTypes.DROPDOWN,
-            options: [{ id: 'empty', name: 'Choose one...' }],
-            value: null
+            expect(field.hasEmptyValue).toBe(true);
+            expect(field.emptyOption).toEqual({ id: 'empty', name: 'Choose one...' });
+            expect(field.options).toEqual([
+                { id: 'empty', name: 'Choose one...' },
+                { id: 'one', name: 'One' }
+            ]);
         });
 
-        expect(field.hasEmptyValue).toBe(true);
-        expect(field.emptyOption).toEqual({ id: 'empty', name: 'Choose one...' });
-    });
-
-    it('should add default "empty" option to the options if hasEmptyValue is true but "empty" option is not present', () => {
-        const field = new FormFieldModel(new FormModel(), {
-            type: FormFieldTypes.DROPDOWN,
-            options: [{ id: 'one', name: 'One' }],
-            value: null,
-            hasEmptyValue: true
+        it('should parse dropdown with multiple options', () => {
+            const field = new FormFieldModel(new FormModel(), {
+                type: FormFieldTypes.DROPDOWN,
+                options: [
+                    { id: 'fake-option-1', name: 'fake label 1' },
+                    { id: 'fake-option-2', name: 'fake label 2' },
+                    { id: 'fake-option-3', name: 'fake label 3' }
+                ],
+                value: [],
+                selectionType: 'multiple'
+            });
+            expect(field.hasMultipleValues).toBe(true);
+            expect(field.hasEmptyValue).toBe(false);
         });
 
-        expect(field.hasEmptyValue).toBe(true);
-        expect(field.emptyOption).toEqual({ id: 'empty', name: 'Choose one...' });
-        expect(field.options).toEqual([
-            { id: 'empty', name: 'Choose one...' },
-            { id: 'one', name: 'One' }
-        ]);
+        describe('should leave not resolved value (in case of delayed options)', () => {
+            it('when string', () => {
+                const field = new FormFieldModel(new FormModel(), {
+                    type: FormFieldTypes.DROPDOWN,
+                    options: [],
+                    value: 'delayed-option-id'
+                });
+
+                expect(field.value).toBe('delayed-option-id');
+            });
+
+            it('when object', () => {
+                const field = new FormFieldModel(new FormModel(), {
+                    type: FormFieldTypes.DROPDOWN,
+                    options: [],
+                    value: { id: 'delayed-option-id', name: 'Delayed option' }
+                });
+                expect(field.value).toBe('delayed-option-id');
+            });
+        });
     });
```

Comment:
Reorganization of dropdown tests - make them similar to radio ones, indeed describe block added and **should parse and leave dropdown value as is** changed to **should leave not resolved value (in case of delayed options)** other tests remained the same